msg/en_US.utf8/utils.msg

```diff
@@ -1309,3 +1310,25 @@ valid options:\n\
                                   for flashback; default: outputs only SQL statements for flashback\n\
     --oldest                      flashback is performed in chronological order for SQL statements executed from start time to end time;\n\
                                   default: latest\n
+
+$set 59 MSGCAT_UTIL_SET_MEMMON
+1 Mutually exclusive options cannot be used together.\n
+2 'memory_monitorting' option is turned off. Please set system parameter 'memory_monitoring' in 'cubrid.conf'.\n
+3 Argument %1$s in option '-m(--module)' does not match in the list of registered server's module.\n
+4 -c(--tran-count) option must be used with -t(--transaction) option\n\
+59 Memmon cannot run as standalone mode.\n
+60 \
+memmon: Provides memory usage information of a server.\n\
+usage: %1$s memmon [OPTION] database-name\n\
+\n\
+valid options:\n\
+    -m, --module=MODULE_NAME      print the module's total memory usage and memory usage details.\n\
+                 MODULE_INDEX\n\
+                 b(or brief)      print summary of memory usage of the entire registered modules.\n\
+    -t, --transaction             print transaction id and memory usage of the entire active transaction.\n\
+    -c, --tran-count=NUMBER       set how many transactions will be printed. If NUMBER is greater than the total\n\
+                                  number of active transactions of the server, memmon will print only the number of\n\
```

Comment:
```
If NUMBER is greater than the total\n\
                                   number of active transactions of the server, memmon will print only the number of\n\
```
Does it mean to print the "number" or as many transactions as possible? It's confusing, actually, the sentence means it prints the number, doesn't it?

Reply:
Need to explain this option have to use together -t option.